heareval/tasks/pipeline.py

```diff
@@ -584,34 +692,38 @@ def run(self):
         for split_path in self.requires()["data"].workdir.iterdir():
             audiodf = pd.DataFrame(
                 [(a.stem, a.suffix) for a in list(split_path.glob("*.wav"))],
-                columns=["slug", "ext"],
+                columns=["unique_filestem", "ext"],
             )
             assert len(audiodf) != 0, f"No audio files found in: {split_path}"
             assert (
-                not audiodf["slug"].duplicated().any()
+                not audiodf["unique_filestem"].duplicated().any()
             ), "Duplicate files in: {split_path}"
+            assert len(audiodf["ext"].drop_duplicates()) == 1
+            assert audiodf["ext"].drop_duplicates().values[0] == ".wav"
```

Comment:
Any reason why these would fail if we are running .glob("*.wav") -- just a sanity check?

Reply:
Just sanity checks